src/lib.rs

```diff
@@ -251,110 +247,116 @@ impl<'a, T> core::clone::Clone for Menu<'a, T> {
 
 impl<'a, T> Runner<'a, T>
 where
-    T: core::fmt::Write,
+    T: embedded_io::Write + embedded_io::Read + embedded_io::ReadReady,
 {
     /// Create a new `Runner`. You need to supply a top-level menu, and a
     /// buffer that the `Runner` can use. Feel free to pass anything as the
     /// `context` type - the only requirement is that the `Runner` can
     /// `write!` to the context, which it will do for all text output.
```

Comment:
Not any more!

Reply:
Technicall I think this is all still accurate. The `context` just needs to be `write!()`-able still, it just uses a different underlying `Write` impl

Reply:
The `only`, presumably.